addons/website/static/src/snippets/s_countdown/000.js

```diff
@@ -30,7 +30,7 @@ const CountdownWidget = publicWidget.Widget.extend({
         this.progressBarWeight = this.el.dataset.progressBarWeight;
 
         this.textColor = this._ensureCssColor(this.el.dataset.textColor);
-        this.layoutBackgroundColor = this._ensureCssColor(this.el.dataset.layoutBackgroundColor);
+        this.layoutBackgroundColor = this._ensureCssColor(this.el.dataset.layoutBackgroundColor) || 'rgba(0, 0, 0, 0)';
```

Comment:
I think this bug is gone since our last colopicker update so no need to add this. Can you test again it plz

Reply:
I have checked in the version sass-13.2 and the bug still exists in the version.

Reply:
I am testing this right now and can not reproduce the bug in runbot saas-13.2

Reply:
> I am testing this right now and can not reproduce the bug in runbot saas-13.2

Here is the video link how to reproduce the bug:  https://youtu.be/BYO6EkXlLkg

Reply:
Oh only on boxes layout ok then.